### ISSUE TITLE
BBC streams not working

### ISSUE THREAD

**Issue by chainsawdave** [RalphBacon/260-ESP32-Web-Pages-Part-2#1]
Hi Ralph Love the channel. Just build an internet radio based on Ed Smallenberg's project and yours. Do you have an uptake on who to get BBC stations? I'm planning to add a bluetooth transmitter then I saw your video. We had the same idea;)

**Comment by RalphBacon** [comment]
Hi chainsawdave! (I know, you’re probably called Mike or something, but not Dave) 😅

So you built the Internet radio?  I’m hoping you don’t live in the UK because the powers-that-be have really done the dirty on us here. Let me explain.

Just (about 1 year later TBH) as I got the Internet Radio all running just fine and dandy, the commercial radio stations (almost all owned and operated by Bauer Media) decided that you had to “log in” after registering with them, of course. There is no way to do that with an Internet Radio (unless you’re using a Raspberry Pi and can get to a browser page). So that killed pretty much all the commercial radio stations I listened to. Sigh.

Then the BBC decided that they would use “chunking” in their streams. I spent some considerable time figuring that all out and got it working very well. I published the updated code on my GitHub with the usual video. The BBC then decided they were withdrawing their .mp3 streams as they were little used (they said. I don’t think they cared one way or the other, they just didn’t want to maintain that method of consuming BBC output). So that removed the BBC radio streams I listened to. Sigh.

Luckily, the one stream I listened to a lot (and now, 24*7) is a German pop music station that has 90% American/UK music and 10% European/German so that’s what I’m listening to right now as I type this 👍 In case you want to check it out, it’s called Antenne 1 at http://antenne1.de but the commentary / ads is all 100% German of course!

If you live in the States I’m pretty sure they do not have such draconian rules and just stream music stations using normal .mnp3 stream. A quick Google of .mp3 (or .aac) streams for New York, NY gives a whole list (eg https://www.wnyc.org/audio/other-formats/ ) so I’m guessing you will be spoiled for choice.

I also expect this to be the case in other parts of the world, not least because they want listeners to actually listen to their output without jumping through hoops, and advertisers will pressure those stations to do this. Just not in the UK, that has this Bauer Media (ironically, a German-owned business empire) monopoly. 🤬

TL;DR No, I never did find a suitable alternative BBC stream to listen to using this Internet Radio. Best bet is using a browser (either PC/MAC or Raspberry Pi) or an old smart phone.

What? Bitter? Me? Not at all! Well, OK, I’m really mad with the way the UK government allowed all this to happen but there is nothing I can do about it other than listen using my tablet or Amazon Echo – which I now do too!

Thanks for your kind words about my channel too, I hope it provides some value to you going forward.
 
Best Regards
Ralph S Bacon
 <http://www.youtube.com/ralphbacon> www.youtube.com/ralphbacon

**Comment by chainsawdave** [comment]
Hi RalphThanks for your reply.I’m actually a David and originally from the United Kingdom but I now live in the US. Have done for over 20 years. I’m a Radio 4 addict so hence the request. Maybe some radio 2 Jerry Vine is an old friend. Happy to send some of the stations that work for me here. The quality is really impressive. I use Alexa as well but your radio is cool. Working on adding BT when your break out board arrives.  I’d like to build a retro radio case for it too. I’m using the VS1053 that can do AcC but can’t seem to get m3u to work. I wonder if there’s a solution with BBC sounds?Keep up the good workCheers David
Sent from my iPhoneOn Jul 10, 2024, at 7:59 AM,

From: chainsawdave ***@***.***> 

Sent: Monday, July 8, 2024 8:46 PM

To: RalphBacon/260-ESP32-Web-Pages-Part-2 ***@***.***>

Cc: Subscribed ***@***.***>

Subject: [RalphBacon/260-ESP32-Web-Pages-Part-2] BBC streams not working (Issue #1)


Hi Ralph Love the channel. Just build an internet radio based on Ed Smallenberg's project and yours. Do you have an uptake on who to get BBC stations? I'm planning to add a bluetooth transmitter then I saw your video. We had the same idea;)



—

**Comment by chainsawdave** [comment]
Hi Ralph
Hope you are well. It's your British follower from the US. I ordered your
breakout board, Works perfectly. If anyone needs any. So, I built a version
of the radio, listening to it right now. I also got the KCM BT to stream to
a BT Speaker. Just wanted you to know it all worked great. All I had to do
was add in a B0505S 5V isolating transformer to remove the ground loop. Now
to work on the case. Just wanted to let you know the progress and how your
videos help.
Cheers
David

**Comment by RalphBacon** [comment]
Hey Dave! (Wow, you really are called Dave, my bad).

Merry Xmas 2024!

It’s great that you got the radio progressed to this stage. 

I really envy you in the USA with all those stations available. Perhaps I should explore more of them to try and find an “Adult Contemporary Music” that encompasses my tastes.

How are you going to build a case? When I built my radio, another kind YT viewer (whom I had helped quite a bit) sent me their 3D printed kit version as I did not have a 3D printer. Now I have two 3D printers! Funny how things move along as time goes by.

Nice that you let me know it’s all working nicely, good job (as the Americans are prone to saying!).

Merry Xmas again, and a Happy New Year too 🎄

Best Regards
Ralph S Bacon

**Comment by chainsawdave** [comment]
Haha yep I'm really Dave!
I've had a 3D printer for a while but just upgraded to the Bambu X1C which
is amazing. I plan to build my case over the holiday, it was going to be a
combination of wood and 3D printing. But we shall see if I have the time.
The front is looking good.I've added a battery monitor and two leds. One
for the BT connection and one for charging (probably go for a
multi-coloured Led (green for on and red for charging). I'm still grumpy
about the BBC streams and have not found a good way around to get many UK
stations other than Heart FM.

I think you will like DixieRock.com  Give it a go.
das-edge09-live365-dal03.cdnstream.com/a88164 let me know.

Have a great Christmas and new year.
Cheers
David
